### PR TITLE
[ISSUE-2758] NullPointerException in QueryTimeManager.checkQueryAlive()

### PR DESCRIPTION
**Pull request by Alima777**
The situation is described in #2758.

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/query/control/QueryTimeManager.java [hunk 1]
```diff
@@ -90,20 +90,18 @@ public AtomicBoolean unRegisterQuery(long queryId) {
...
-        });
-    queryScheduledTaskMap.computeIfPresent(
-        queryId,
-        (k, v) -> {
-          queryScheduledTaskMap.get(queryId).cancel(false);
+          ScheduledFuture<?> scheduledFuture = queryScheduledTaskMap.remove(queryId);
+          if (scheduledFuture != null) {
+            scheduledFuture.cancel(false);
+          }
           return null;
```
Comment (wangchao316) [inline server/src/main/java/org/apache/iotdb/db/query/control/QueryTimeManager.java:97]: maybe return null.

File: server/src/main/java/org/apache/iotdb/db/query/control/QueryTimeManager.java [hunk 2]
```diff
@@ -90,20 +90,18 @@ public AtomicBoolean unRegisterQuery(long queryId) {
...
         });
     return successRemoved;
   }
 
   public static void checkQueryAlive(long queryId) {
-    if (getInstance().queryInfoMap.get(queryId) != null
-        && getInstance().queryInfoMap.get(queryId).isInterrupted()) {
+    QueryInfo queryInfo = getInstance().queryInfoMap.get(queryId);
+    if (queryInfo != null && queryInfo.isInterrupted()) {
       if (getInstance().unRegisterQuery(queryId).get()) {
```
Comment (wangchao316) [inline server/src/main/java/org/apache/iotdb/db/query/control/QueryTimeManager.java:105]: hi,   getInstance().unRegisterQuery(queryId) maybe return null, so #get() will throw nullpointerexception

Comment (Alima777) [reply server/src/main/java/org/apache/iotdb/db/query/control/QueryTimeManager.java:105]: It may return false but never return null. Am i wrong?

Comment (Alima777) [reply server/src/main/java/org/apache/iotdb/db/query/control/QueryTimeManager.java:105]: It may return false but never return null. Am i wrong?

**Comment by qiaojialin** [comment]
add more description and resubmit

File: server/src/main/java/org/apache/iotdb/db/query/control/QueryTimeManager.java (hunk 2 above)
Comment (wangchao316) [reply server/src/main/java/org/apache/iotdb/db/query/control/QueryTimeManager.java:105]: sorry, I see fault..   map k-v return null...  
unRegisterQuery return boolean..
you are right...

File: server/src/main/java/org/apache/iotdb/db/query/control/QueryTimeManager.java [hunk 3]
```diff
@@ -90,20 +90,18 @@ public AtomicBoolean unRegisterQuery(long queryId) {
         queryId,
         (k, v) -> {
           successRemoved.set(true);
-          return null;
-        });
-    queryScheduledTaskMap.computeIfPresent(
-        queryId,
-        (k, v) -> {
-          queryScheduledTaskMap.get(queryId).cancel(false);
+          ScheduledFuture<?> scheduledFuture = queryScheduledTaskMap.remove(queryId);
```
Comment (wangchao316) [inline server/src/main/java/org/apache/iotdb/db/query/control/QueryTimeManager.java:93]: remove queryId from Map, this is solution..
good...

**Review by wangchao316 — APPROVED** [review]

**Pull request merged**